src/dotnet-install.ps1

```diff
@@ -429,16 +442,16 @@ function Get-Latest-Version-Info([string]$AzureFeed, [string]$Channel) {
 
     $VersionFileUrl = $null
     if ($Runtime -eq "dotnet") {
-        $VersionFileUrl = "$UncachedFeed/Runtime/$Channel/latest.version"
+        $VersionFileUrl = "$AzureFeed/Runtime/$Channel/latest.version"
```

Comment:
The $UncachedFeed doesn't actually exist anymore. AzureFeed is set to the uncached feed if -NoCdn is set.
I don't think dotnetbuilds will ever have the latest.version stuff, because that is from 3.1 builds, and dotnetbuilds will only be used for 6.0 and later builds.

If we really need to skip the cdn for caching reasons to ensure we get the absolute latest, I would rather we set caching rules for these files in the cdn itself. We can tell the cdn to skip caching for files like this.

Reply:
@bekir-ozturk  could you please check above?
I'm fine with change unless we specifically opted in to use uncached feed here due to known cache issues.